components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('SchoolCard', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
+      <SchoolCard
+        cardFlipCallback={() => {}}
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="source"
+        name="school name"
+        website="website"
+      >
+        Test
+      </SchoolCard>,
+    );
+  });
+
+  it('should render with required props', () => {
+    createShallowSnapshotTest(
+      <SchoolCard
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="logoSource.com"
+        name="school name"
+        website="website.com"
+      >
+        <p>Testing!</p>
+      </SchoolCard>,
+    );
+  });
+
+  it('call to showBackOfCard/showFrontOfTheCard flips isFrontOfCardShowing', () => {
```

Comment:
Let's name this a little differently. I suggest: `should flip card when locations button is clicked`

Additionally, instead of just testing that state changes when the function is called, what if we tested the changes after simulating the actual interaction that calls the function? (See CloseButton tests)